WP_Auth0.php

```diff
@@ -123,6 +123,10 @@ public function init() {
 
 		$this->check_signup_status();
 
+		if ( $this->a0_options->get( 'auto_login' ) ) {
+			WP_Auth0_Nonce_Handler::getInstance()->setCookie();
```

Comment:
Is this well indented?

Reply:
Like, correct whitespace? Yes. This project mixes tabs and spaces, as well as how many spaces for each. I'll do an automated whitespace cleanup before I start the next release.